tests/subsys/bluetooth/fast_pair/storage/account_key_storage/src/main.c

```diff
@@ -18,6 +18,7 @@
 
 #define ACCOUNT_KEY_MAX_CNT	CONFIG_BT_FAST_PAIR_STORAGE_ACCOUNT_KEY_MAX
 
+BUILD_ASSERT(!IS_ENABLED(CONFIG_BT_FAST_PAIR_STORAGE_AK_BOND));
```

Comment:
Consider commenting that it's because the AK bond storage feature is not yet supported by the unit tests (to clarify why the assert was introduced here)

Reply:
Done.